hack/install_bats.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+die() { echo "${1:-No error message given} (from $(basename $0))"; exit 1; }
+
+buildDir=$(mktemp -d)
+git clone https://github.com/bats-core/bats-core $buildDir
+
+pushd $buildDir
+echo `pwd`
```

Comment:
nit (minor, no need to fix): it's cleaner to just say `pwd`

Reply:
Thanks, fixed now